Web/starter_project_vanguard/components/profile/AccountSetting.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import { AiFillEyeInvisible, AiFillEye } from 'react-icons/ai'
+
+const AccountSetting: React.FC = () => {
+  const [showCurrPassword, setShowCurrPassword] = useState<boolean>(false)
```

Comment:
[nit] better name it `showCurrentPassword` instead of `showCurrPassword`
we wouldn't be saving much space